src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java

```diff
@@ -57,19 +52,19 @@ public final class CaseEndpoint implements CTPEndpoint {
   private CaseService caseService;
   private CaseGroupService caseGroupService;
   private CategoryService categoryService;
-  private CaseRepository caseRepository;
+  private CaseEventRepository caseEventRepository;
 
   /** Contructor for CaseEndpoint */
   @Autowired
   public CaseEndpoint(
       final CaseService caseService,
       final CaseGroupService caseGroupService,
       final CategoryService categoryService,
-      final CaseRepository caseRepository) {
+      final CaseEventRepository caseEventRepository) {
     this.caseService = caseService;
     this.caseGroupService = caseGroupService;
     this.categoryService = categoryService;
-    this.caseRepository = caseRepository;
+    this.caseEventRepository = caseEventRepository;
```

Comment:
Shouldn't need any of the above `caseEventRepository` if this POC works as expected